src/components/History/NetworkUpgradeSummary.tsx

```diff
@@ -0,0 +1,131 @@
+// Libraries
+import React from "react"
+import styled from "styled-components"
+import { FormattedNumber } from "react-intl"
```

Comment:
As we're using `react-intl` now, I used FormattedNumber here instead of [the common approach we're using elsewhere](https://github.com/ethereum/ethereum-org-website/blob/43c8b5709da532247c5f7668da25d56daf42e384/src/components/StatsBoxGrid.tsx#L344) as it seemed a lot cleaner. 

Now noticing we don't handle Farsi numbers by doing it this way.